tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5875,7 +5876,7 @@ declare namespace ts {
         getNameOrDottedNameSpan(fileName: string, startPos: number, endPos: number): TextSpan | undefined;
         getBreakpointStatementAtPosition(fileName: string, position: number): TextSpan | undefined;
         getSignatureHelpItems(fileName: string, position: number, options: SignatureHelpItemsOptions | undefined): SignatureHelpItems | undefined;
-        getRenameInfo(fileName: string, position: number, options?: RenameInfoOptions): RenameInfo;
+        getRenameInfo(fileName: string, position: number, preferences: UserPreferences): RenameInfo;
```

Comment:
It feels like we could keep the old signature as a deprecated overload if we wanted to avoid the API break.